src/generators/integer/index.test.ts

```diff
@@ -0,0 +1,61 @@
+import {Chance} from 'chance';
+
+import createIntegerGenerator from '.';
+
+describe('integer', () => {
```

Comment:
Can you remove the silly integer and date tests from `src/index.test.ts` now that we have these?

Reply:
Removed in 169af8b